### DISCUSSION TITLE
Install alpha-nvim problems in neovim using vim-plug

### DISCUSSION THREAD

**Discussion by CuiEM** [goolord/alpha-nvim#245] [Q&A]
here is my error display

       Error detected while processing /home/cuiem/.config/nvim/init.vim:
       line   10:

        E5108: Error executing lua [string ":lua"]:1: module 'alpha' not found:

        no field package.preload['alpha']
        no file './alpha.lua'
        no file '/usr/share/luajit-2.1.0-beta3/alpha.lua'
        no file '/usr/local/share/lua/5.1/alpha.lua'
        no file '/usr/local/share/lua/5.1/alpha/init.lua'
        no file '/usr/share/lua/5.1/alpha.lua'
        no file '/usr/share/lua/5.1/alpha/init.lua'
        no file './alpha.so'
        no file '/usr/local/lib/lua/5.1/alpha.so'
        no file '/usr/lib/x86_64-linux-gnu/lua/5.1/alpha.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
        stack traceback:
        [C]: in function 'require'
        [string ":lua"]:1: in main chunk

here is my code in init.vim

      call plug#begin('~/.local/share/nvim/site/plugged')  
       2 Plug 'bling/vim-airline'
       3 Plug 'jiangmiao/auto-pairs'
       4 Plug 'folke/tokyonight.nvim'
       5 Plug 'scrooloose/nerdtree'
       6 Plug 'goolord/alpha-nvim'
       7 Plug 'kyazdani42/nvim-web-devicons'
       8 lua << EOF
       9 require'alpha'.setup( require'alpha.themes.startify'.config, require'nvim-web-devicons'.setup() )
      10 EOF
      11  
      12 Plug 'lervag/vimtex'
      13 let g:tex_flavor='latex'
      14 let g:vimtex_view_general_viewer = 'zathura'
      15 let g:vimtex_view_method='zathura'
      16 let g:vimtex_quickfix_mode=0
      17 set conceallevel=1
      18 let g:tex_conceal='abdmg'
      19  
      20 call plug#end()
      21  
      22 syntax on
      23  
      24 set nu
      25 set cursorline
      26 set autoindent
      27 set showmatch
      28 set smartindent
      29 set encoding=utf-8
      30  
      31 let g:auto_save = 1
      32  
      33 " ###################### 设置主题外观 ####################
      34 set background=dark
      35 set cursorline
      36 set cursorcolumn
      37 colorscheme tokyonight

**Comment by goolord** [comment]
i don't use vimplug but my guess is that you should move the configuration outside of `plug#end`

**Reply by CuiEM** [reply]
thx! but after i moved the configuration outside of plug#end, a new problem showed out:


      Error detected while processing CursorMoved Autocommands for "<buffer=1>":
      Error executing lua callback: Vim:E117: Unknown function: virtcol2col
      stack traceback:
        [C]: in function 'virtcol2col'
        .../.local/share/nvim/site/plugged/alpha-nvim/lua/alpha.lua:648: in function 'move_cursor'
        .../.local/share/nvim/site/plugged/alpha-nvim/lua/alpha.lua:527: in function <.../.local/share/nvim/site/plugged/alpha-nvim/lua/alpha.lua:526>

even though there is a problem, i can still see the alpha-start in neovim like this:


                                     __
         ___     ___    ___   __  __ /\_\    ___ ___
         / _ `\  / __`\ / __`\/\ \/\ \\/\ \  / __` __`\
         /\ \/\ \/\  __//\ \_\ \ \ \_/ |\ \ \/\ \/\ \/\ \
         \ \_\ \_\ \____\ \____/\ \___/  \ \_\ \_\ \_\ \_\
         \/_/\/_/\/____/\/___/  \/__/    \/_/\/_/\/_/\/_/

         [e] New file

         MRU /home/cuiem/.config/nvim

         [0]   init.vim

         MRU

         [10]   /usr/share/texlive/texmf-dist/tex/latex/ctex/fontset/ctex-fontset-fandol.def
         [11]   ~/.config/nvim/init.vim

         [q] Quit

it confused me that why they're not in the center of my screen but more likely in the left. 
Also there are few unrecognizable code.

i'm totally new to the neovim, i really don't know how to deal with those problems. thank you very much!
